src/custom/state/claim/hooks/index.ts

```diff
@@ -288,17 +304,28 @@ function useDeploymentTimestamp(): number | null {
   const { chainId } = useActiveWeb3React()
   const vCowContract = useVCowContract()
   const [timestamp, setTimestamp] = useState<number | null>(null)
+  const [isMounted, setIsMounted] = useState(true)
```

Comment:
I think you should use a `useRef` instead.
Otherwise the isMounted flag will cause the issue you are trying to prevent here.

Reply:
Good suggestion, looks like its working great.